test/ngAnimate/animateCssSpec.js

```diff
@@ -1069,6 +1069,36 @@ describe("ngAnimate $animateCss", function() {
           expect(count.normal).toBe(3);
         }));
 
+        it("should cache the post-quiet state detection and flush one frame after the animation is complete",
+          inject(function($animateCss, $document, $rootElement, $$rAF) {
+
+          var i, animator, elms = [];
+          for (i = 0; i < 5; i++) {
+            var elm = jqLite('<div>' + i + '</div>');
+            $rootElement.append(elm);
+            animator = $animateCss(elm, { to: {color:'green'}, duration: 0.5 });
+            var runner = animator.start();
+            elms.push(elm);
+          }
+
+          expect(count.normal).toBe(2); //first + stagger
+          triggerAnimationStartFrame();
+
+          expect(count.normal).toBe(3); //first + stagger + post-quiet
```

Comment:
Do you by any chance mean `quit` (here and elsewhere) ?
(If not, I wonder what `post-quiet` is about.)